File: Auto-generated-tasks/Difficulty-3/Task-2/Solution.go
package "main"
import "fmt"
Count := 6
i1 := 0
var S[Count] string
for i1 = 0; i1 < Count; i1++ {
	S[i1] = "No"
	S[i1 + 1] = "Buf451"
	S[i1 + 2] = "k719"
}
for i1 = 0; i1 < Count; i1++ {
	fmt.PrintLn("S[", i1, "]=", S[i1])
}

File: Auto-generated-tasks/Difficulty-3/Task-2/Solution.f90
integer:: Count = 6, i1
Character(len=6)(dimension=Count):: S
do i1 = 0, Count
	S(i1) = "No"
	S(i1 + 1) = "Buf451"
	S(i1 + 2) = "k719"
end do
do i1 = 0, Count
	print *,"S[", i1, "]=", S(i1)
end do